spring-cloud-gcp-pubsub/src/main/java/org/springframework/cloud/gcp/pubsub/integration/inbound/PubSubInboundChannelAdapter.java

```diff
@@ -56,7 +56,7 @@
 
 	private HeaderMapper<Map<String, String>> headerMapper = new PubSubHeaderMapper();
 
-	private Class payloadType = byte[].class;
+	private Class<?> payloadType = byte[].class;
```

Comment:
With this you have missed to update Copyright.
Or are you going to use your magic script? 😄

Reply:
Apparently, having a magic script is insufficient if one forgets to run it :|